docs/release_runbook.md

```diff
@@ -0,0 +1,34 @@
+# Release
+The goal of this document is to describe the release process on repos that implements the [trigger_prerelease](../.github/workflows/trigger_prerelease.yaml) workflow.
+
+## Pre-Releases
+
+**Important Note**: Creating releases from the GH interface is not longer the recommended way. In case there is a strong reason to do it please follow the [Emergency/By-pass Release](#emergencyby-pass-release)
+
+To have some context before releasing, the following steps of the process are automated by the [release toolkit](https://github.com/newrelic/release-toolkit#readme):
+* `CHANGELOG.md` update: A new entry will be added based on unreleased entries and dependencies commits. [Details](https://github.com/newrelic/release-toolkit#render-markdown-and-update-markdown).
+* GH Pre-Release creation: A GH Pre-Release is generated with Notes taken from the changelog.
+* Release version calculation: Is calculated based on the Changelog entries. [Details](https://github.com/newrelic/release-toolkit#next-version).
+
+You can check locally how the the outcomes of the release toolkit will look like by running [these steps](https://github.com/newrelic/release-toolkit/tree/main/contrib/ohi-release-notes#use-script-locally), or the make target `rt-update-changelog` if exist in the repo.
+
+There are two ways to create pre-releases:
+
+- Automatically: Scheduled weekly.
+- Manually: Triggered from the GH Actions interface (workflow dispatch)
+
+### Held releases (manual and automatic)
+
+There are two mechanism to stop the release creation:
+* Global block: This will stop any automatic release based on the [trigger_prerelease](../.github/workflows/trigger_prerelease.yaml) workflow, to be use for reasons like a code freeze. Follow [this instructions](../README.md/#automatic-releases-block-endpoint) to enable/disable this.
+* Local held: By adding the `## Held` header in `CHANGELOG.md`. [Details](https://github.com/newrelic/release-toolkit#automated-releasing).
+
+### Emergency/By-pass Release
```

Comment:
🤔 Do you have an emergency scenario in mind? Using the workflow-dispatch in GH actions interface is not only safer but should also be faster, isn't it?

Reply:
> 🤔 Do you have an emergency scenario in mind? 
Not that I can imagine, in fact for the prometheus configurator this was not even a choice. 
But the given that it is possible I prefer it to have it documented.

Reply:
It makes sense, thanks! I think the _Emergency_ word confused me. It made me think this is the way to go if some sort of emergency release needs to be done.